napari/_vispy/layers/image.py

```diff
@@ -93,6 +93,18 @@ def __init__(
             layer_node_class=layer_node_class,
         )
 
+        self._array_like = True
+
+        # set the 3D shader step size bounds
+        self.min_step_size = 0.05
```

Comment:
```suggestion
        self.min_step_size = 0.1
```
Looks like this is limited by vispy to 0.1:
https://github.com/vispy/vispy/blob/3e2aadacdc320fffa6b251d96de3c75ceed76065/vispy/visuals/volume.py#L1166-L1172